app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+
+  USER_CATEGORY = [
+    ["manager", "Manager"],
+    ["developer", "Developer"],
+    ["qa", "QA"],
+  ]
+
+  enum user_category: {
+    manager: 0,
+    developer: 1,
+    qa: 2
+  }
```

Comment:
We can make this more readable like this
```suggestion
  USER_CATEGORIES = { manager: 0, developer: 1, qa: 2 }.freeze
  enum user_category: USER_CATEGORIES
```

Reply:
I think by doing this. We are not improving the readability but better memory consumption